src/style.css

```diff
@@ -1,3 +1,63 @@
+html {
+  margin: 0;
+  padding: 0;
+  --alternative-text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: block;
+  cursor: pointer;
+  color: black;
```

Comment:
it's better use hex values instead of reserved words

Reply:
✅